### PR TITLE
jankyborders service option order above or below

### PR DESCRIPTION
**Pull request by aspauldingcode**
Adds undocumented JankyBorders option `order` which can be set to `above` or `below`. Defaults to `below` because it causes issues for some users (hence the reason for not being documented). Helpful for covering the window frame on macos.

### CODE REVIEW

**Comment by Enzime** [comment]
Can you squash the commits into one commit?

**Comment by aspauldingcode** [comment]
> Can you squash the commits into one commit?

There, I think I did it with [`af95f7b` (#1095)](https://github.com/LnL7/nix-darwin/pull/1095/commits/af95f7b7ec80811cd5662d6b08b45f0160c85d1d)

**Comment by Enzime** [comment]
Can you also update the jankyborders test?

File: modules/services/jankyborders/default.nix [hunk 1]
```diff
@@ -80,6 +80,15 @@ in {
       '';
     };
 
+    order = mkOption {
+      type = types.str;
```
Comment (Enzime) [inline modules/services/jankyborders/default.nix:84]:
```suggestion
      type = types.enum [ "above" "below" ];
```

**Review by Enzime — CHANGES_REQUESTED** [review]

**Comment by aspauldingcode** [comment]
Hm okay I think I did it. 

> update the jankyborders test

**Comment by Enzime** [comment]
Did you forget to push the changes for the tests? I don't see any new changes

**Comment by aspauldingcode** [comment]
> forget to push

Yes.. no.. yes.. maybe. oops.

I think I did this correctly?

**Review by Enzime — APPROVED** [review]
LGTM

Thanks for the contribution

**Comment by aspauldingcode** [comment]
Hooray, my very first pull request (to any project, ever in my life) being merged. I will remember this moment!

**Pull request merged**